pkg/cmd/citadel/options.go

```diff
@@ -48,7 +48,7 @@ var (
 )
 
 func init() {
-	flag.StringVar(&args.endpoint, "endpoint", "", `the listen address (ex. unix:///tmp/kms.sock)`)
+	flag.StringVar(&args.endpoint, "endpoint", "", `the listen address (ex. unix:///tmp/socketfile.sock)`)
```

Comment:
This undoes my previous change. I'm trying to keep doc strings short (for an 80 character terminal). Also, "socketfile.sock" is redundant.

Reply:
Trying to remove the use of "kms", and this was what the kube examples used.  I can pick something else that is short.